src/main/java/org/apache/commons/lang3/math/NumberUtils.java

```diff
@@ -1013,6 +1021,7 @@ public static long min(final long... array) {
         validateArray(array);
 
         // Finds and returns min
+        @SuppressWarnings("index:array.access.unsafe.high.constant") // validateArray(array) => array.length != 0
```

Comment:
Why not annotate `validateArray` as `@EnsuresMinLenIf`?

Reply:
`validateArray` does not return a boolean type, hence I cannot annotate it with `@EnsuresMinLenIf`

Reply:
In that case, you should probably just annotate the argument to this method as `@MinLen(1)`, since you should [annotate normal behavior](https://checkerframework.org/manual/#annotate-normal-behavior).